integration/testdata/conda-spdx.json.golden

```diff
@@ -1,113 +1 @@
-    },
-    {
-      "spdxElementId": "SPDXRef-Application-ee5ef1aa4ac89125",
-      "relatedSpdxElement": "SPDXRef-Package-b1088cb4090e3a55",
-      "relationshipType": "CONTAINS"
-    },
-    {
-      "spdxElementId": "SPDXRef-Application-ee5ef1aa4ac89125",
-      "relatedSpdxElement": "SPDXRef-Package-6b677e82217fb5bd",
-      "relationshipType": "CONTAINS"
-    }
-  ]
-}
\ No newline at end of file
+{"spdxVersion":"SPDX-2.3","dataLicense":"CC0-1.0","SPDXID":"SPDXRef-DOCUMENT","name":"testdata/fixtures/fs/conda","documentNamespace":"http://aquasecurity.github.io/trivy/filesystem/testdata/fixtures/fs/conda-567b8a08-dba6-4dfc-9dac-d1f357f80b02","creationInfo":{"licenseListVersion":"","creators":["Organization: aquasecurity","Tool: trivy-dev"],"created":"2023-05-15T09:49:59Z"},"packages":[{"name":"conda-pkg","SPDXID":"SPDXRef-Application-ee5ef1aa4ac89125","downloadLocation":"NONE","sourceInfo":"Conda","copyrightText":"","primaryPackagePurpose":"APPLICATION"},{"name":"openssl","SPDXID":"SPDXRef-Package-b1088cb4090e3a55","versionInfo":"1.1.1q","supplier":"NOASSERTION","downloadLocation":"NONE","licenseConcluded":"OpenSSL","licenseDeclared":"OpenSSL","copyrightText":"","externalRefs":[{"referenceCategory":"PACKAGE-MANAGER","referenceType":"purl","referenceLocator":"pkg:conda/openssl@1.1.1q"}],"primaryPackagePurpose":"LIBRARY","files":[{"fileName":"miniconda3/envs/testenv/conda-meta/openssl-1.1.1q-h7f8727e_0.json","SPDXID":"SPDXRef-File-600e5e0110a84891","checksums":[{"algorithm":"SHA1","checksumValue":"237db0da53131e4548cb1181337fa0f420299e1f"}],"copyrightText":""}]},{"name":"pip","SPDXID":"SPDXRef-Package-6b677e82217fb5bd","versionInfo":"22.2.2","supplier":"NOASSERTION","downloadLocation":"NONE","licenseConcluded":"MIT","licenseDeclared":"MIT","copyrightText":"","externalRefs":[{"referenceCategory":"PACKAGE-MANAGER","referenceType":"purl","referenceLocator":"pkg:conda/pip@22.2.2"}],"primaryPackagePurpose":"LIBRARY","files":[{"fileName":"miniconda3/envs/testenv/conda-meta/pip-22.2.2-py38h06a4308_0.json","SPDXID":"SPDXRef-File-7eb62e2a3edddc0a","checksums":[{"algorithm":"SHA1","checksumValue":"a6a2db7668f1ad541d704369fc66c96a4415aa24"}],"copyrightText":""}]},{"name":"testdata/fixtures/fs/conda","SPDXID":"SPDXRef-Filesystem-6e0ac6a0fab50ab4","downloadLocation":"NONE","copyrightText":"","attributionTexts":["SchemaVersion: 2"],"primaryPackagePurpose":"SOURCE"}],"relationships":[{"spdxElementId":"SPDXRef-DOCUMENT","relatedSpdxElement":"SPDXRef-Filesystem-6e0ac6a0fab50ab4","relationshipType":"DESCRIBES"},{"spdxElementId":"SPDXRef-Filesystem-6e0ac6a0fab50ab4","relatedSpdxElement":"SPDXRef-Application-ee5ef1aa4ac89125","relationshipType":"CONTAINS"},{"spdxElementId":"SPDXRef-Application-ee5ef1aa4ac89125","relatedSpdxElement":"SPDXRef-Package-b1088cb4090e3a55","relationshipType":"CONTAINS"},{"spdxElementId":"SPDXRef-Application-ee5ef1aa4ac89125","relatedSpdxElement":"SPDXRef-Package-6b677e82217fb5bd","relationshipType":"CONTAINS"}]}
```

Comment:
Can we format this golden file?

Reply:
I created #4404 for that.

Reply:
I have updated this file in https://github.com/aquasecurity/trivy/pull/4380/commits/b15b5b167303c3bbda1a4b21b37f898622dbf2c6